cpp/src/parquet/stream_writer.h

```diff
@@ -151,6 +151,23 @@ class PARQUET_EXPORT StreamWriter {
   StreamWriter& operator<<(const std::string& v);
   StreamWriter& operator<<(::std::string_view v);
 
+  /// \brief Helper class to write variable length raw data.
```

Comment:
How about using `std::basic_string_view<uint8_t>` instead? Perhaps we don't need to create yet another string-view-like class.

Reply:
One compiler seems unhappy about it: https://github.com/viveris/arrow/actions/runs/16043325816/job/45268982458#step:7:1068

Is there a way to fix that?